src/diffusers/loaders/lora_pipeline.py

```diff
@@ -92,7 +93,9 @@ def load_lora_weights(
             pretrained_model_name_or_path_or_dict = pretrained_model_name_or_path_or_dict.copy()
 
         # First, ensure that the checkpoint is a compatible one and can be successfully loaded.
-        state_dict, network_alphas = self.lora_state_dict(pretrained_model_name_or_path_or_dict, **kwargs)
+        state_dict, network_alphas, metadata = self.lora_state_dict(
```

Comment:
@bghira you will need to update the calls to `self.lora_state_dict()` like here:

https://github.com/huggingface/diffusers/blob/65e30907b5df1338ae65a20b78866c87e061c952/examples/dreambooth/train_dreambooth_lora_sdxl.py#L1263

to have `return_metadata=True` and pass that metadata to `save_lora_weights()` as shown in the tests. LMK if this is unclear.

Reply:
https://github.com/huggingface/diffusers/pull/9143#issuecomment-2282779195

Reply:
i was looking for this 🤣